client/index.html

```diff
@@ -1,13 +1,19 @@
 <!DOCTYPE html>
-<html>
+<html ng-app="myApp">
   <head>
     <meta charset="utf-8">
     <title>TimeShare</title>
+    <script src="app/bower_components/angular/angular.min.js"></script>
+    <script src="app/bower_components/angular-ui-router/release/angular-ui-router.min.js"></script>
```

Comment:
script tags should be included in the body. include any link tags or meta tags in the head